### DISCUSSION TITLE
Thoughts on my Mortgage Calculator

### DISCUSSION THREAD

**Discussion by PhysethLeav** [001138477/Tafe2022S213#6] [Q&A]
[](url)
![Screenshot 2022-09-20 202501](https://user-images.githubusercontent.com/111669645/191240212-68ad593c-d86a-4e4c-86c8-4b0b877bb84b.jpg)

**Comment by PhysethLeav** [comment]
Is the title to big? anything you suggest i change before i push to the Devlopment branch?

**Reply by 001138477** [reply]
Looks good, you can push now.
